QualityControl/public/common/filters/filterViews.js

```diff
@@ -51,24 +53,51 @@ const createFilterElement = (config, filterMap, onInputCallback, onEnterCallback
  * @returns {vnode} - virtual node element
  */
 export function filtersPanel(filterModel, viewModel) {
-  const { filterMap, setFilterValue, filterService, isVisible, clearFilter } = filterModel;
+  const {
+    filterMap,
+    setFilterValue,
+    filterService,
+    clearFilter,
+    isRunModeActivated,
+    runNumber,
+    runStatus,
+    isVisible,
+  } = filterModel;
   const onInputCallback = setFilterValue.bind(filterModel);
   const onChangeCallback = setFilterValue.bind(filterModel);
   const onEnterCallback = () => filterModel.triggerFilter(viewModel);
   const clearFilterCallback = clearFilter.bind(filterModel, viewModel);
-  const filtersList = filtersConfig(filterService);
-
-  if (!isVisible || filterModel.inRunMode) {
+  if (!isVisible) {
     return null;
   }
+  const filtersList = isRunModeActivated
+    ? runModeFilterConfig()
+    : filtersConfig(filterService);
 
   return h(
-    '.w-100.flex-row.p2.g2.justify-center#filterElement',
+    '.w-100.flex-column.p2.g2.justify-center#filterElement',
     [
-      triggerFiltersButton(onEnterCallback, filterModel, viewModel),
-      clearFiltersButton(clearFilterCallback),
-      ...filtersList.map((filter) =>
-        createFilterElement(filter, filterMap, onInputCallback, onEnterCallback, onChangeCallback)),
+      h('.flex-row.g2.justify-center', [
+        runModeCheckbox(filterModel, viewModel),
+
+        isRunModeActivated
```

Comment:
This can simply be re-writte as :
```js
      h('.flex-row.g2.justify-center', [
        runModeCheckbox(filterModel, viewModel),
        triggerFiltersButton(onEnterCallback, filterModel),
        ...filtersList.map((filter) =>
          createFilterElement(filter, filterMap, onInputCallback, onEnterCallback, onChangeCallback)),
        isRunModeActivated && clearFiltersButton(clearFilterCallback),
      ]),
```
I know the update button is on the righ ton RunMode but to be consistent, I would say keep it on the left as the normal filter. This also improved code readability